package.json

```diff
@@ -60,7 +60,7 @@
     "moment-duration-format": "^2.3.2",
     "moment-timezone": "^0.5.33",
     "node-sass": "^7.0.1",
-    "openstack-uicore-foundation": "4.1.18",
+    "openstack-uicore-foundation": "4.1.21",
```

Comment:
seems that we need to upgrade this to .22 ?
https://github.com/OpenStackweb/openstack-uicore-foundation/commits/main
also there is a missing commit on uicore ( version 4.1.22)